<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetConfiguration</class>
 <widget class="QDialog" name="GetConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Escoge las claves que quieres obtener:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>AuthorizeRemoteTxRequests</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>ClockAlignedDataInterval</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>ConnectionTimeOut</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>ConnectorPhaseRotation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string>GetConfigurationMaxKeys</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="text">
      <string>HeartbeatInterval</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_7">
     <property name="text">
      <string>LocalAuthorizeOffline</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_8">
     <property name="text">
      <string>LocalPreAuthorize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_9">
     <property name="text">
      <string>MeterValuesAlignedData</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_10">
     <property name="text">
      <string>MeterValuesSampledData</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_11">
     <property name="text">
      <string>MeterValueSampleInterval</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_12">
     <property name="text">
      <string>NumberOfConnectors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_13">
     <property name="text">
      <string>ResetRetries</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_14">
     <property name="text">
      <string>StopTransactionOnEVSideDisconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_15">
     <property name="text">
      <string>StopTransactionOnInvalidId</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_16">
     <property name="text">
      <string>StopTxnAlignedData</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_17">
     <property name="text">
      <string>StopTxnSampledData</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_18">
     <property name="text">
      <string>SupportedFeatureProfiles</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_19">
     <property name="text">
      <string>TransactionMessageAttempts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_20">
     <property name="text">
      <string>TransactionMessageRetryInterval</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_21">
     <property name="text">
      <string>UnlockConnectorOnEVSideDisconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Enviar operaci√≥n</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
